From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Beckett Normington <beckett@b0ba.dev>
Date: Tue, 25 Jul 2023 17:13:12 -0400
Subject: [PATCH] Makefile: Don't run `lib/unicore/mktables` with -maketest

This argument causes the interpreter to hang and consume huge amounts
of RAM.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 4b80079f22738918822df9c30d6be15f5b5b6078..0a319af97e896d5d2784906fd21367e90c293991 100644
--- a/Makefile
+++ b/Makefile
@@ -302,7 +302,7 @@ dist/Unicode-Normalize/Makefile: lib/unicore/CombiningClass.pl
 pod/perluniprops.pod: lib/unicore/CombiningClass.pl
 
 lib/unicore/CombiningClass.pl: $(CONFIGPM)
-	./miniperl_top lib/unicore/mktables -w -C lib/unicore -P pod -maketest -makelist -p
+	./miniperl_top lib/unicore/mktables -w -C lib/unicore -P pod -makelist -p
 
 # The following rules ensure that modules listed in mkppport.lst get
 # their ppport.h installed. There's a bunch of comments in that file,
